plugins/generator-1.20.6/neoforge-1.20.6/generator.yaml

```diff
@@ -1,5 +1,5 @@
 name: NeoForge for @minecraft (@buildfileversion)
-status: dev
+status: stable

```

Comment:
This should be separate PR after I and other potential contributors finish with in-depth testing of finalized genrator

Reply:
:heavy_check_mark: Fixed